README.md

```diff
@@ -36,9 +36,11 @@ In order to see the nice font as we used in the demo, add the following rule in
 ## Workshop steps
 - step 0: install Meteor and test locally
 - step 1: setup the Tic tac toe project
+  - copy the content of the css in the css file generated in your folder
   - you can rename the template in your `.html` file, for example `<template name="gameboard"></template>` instead of the `hello` template
-  - begin with `<div class="gameboard"></div>` (Now you should see a nice green schoolboard)
-  - you can add `<div class="box"></div>` into the `.gameboard` in your template
+  - begin with `<div class="gameboard"></div>` in this template(Now you should see a nice green schoolboard)
+  - you can add `<div class="box"></div>` into the `gameboard` div
+  - Tip: with F12 you can open the developers tools of your browser. And if you use console.log in the js file you can print to the console.
```

Comment:
On WIndows, you can open the dev tools of Chrome. On a Mac, right-click in the browser and choose `inspect`, then select the console tab.